README.md

```diff
@@ -1,13 +1,17 @@
 # AsyncHTTPClient Transport for Swift OpenAPI Generator
 
+[![](https://img.shields.io/badge/docc-read_documentation-blue)](https://swiftpackageindex.com/swift-server/swift-openapi-async-http-client/documentation)
+[![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2Fswift-server%2Fswift-openapi-async-http-client%2Fbadge%3Ftype%3Dswift-versions)](https://swiftpackageindex.com/swift-server/swift-openapi-async-http-client)
+[![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2Fswift-server%2Fswift-openapi-async-http-client%2Fbadge%3Ftype%3Dplatforms)](https://swiftpackageindex.com/swift-server/swift-openapi-async-http-client)
+
 A client transport that uses the [HTTPClient](https://swift-server.github.io/async-http-client/docs/current/AsyncHTTPClient/Classes/HTTPClient) type from the [AsyncHTTPClient](https://github.com/swift-server/async-http-client) library to perform HTTP operations.
```

Comment:
AHC is on the swift package index as well

```suggestion
A client transport that uses the [HTTPClient](https://swiftpackageindex.com/swift-server/async-http-client/1.19.0/documentation/asynchttpclient/httpclient) type from the [AsyncHTTPClient](https://github.com/swift-server/async-http-client) library to perform HTTP operations.
```

Reply:
Good catch, fixed